### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by ad9311**
# Greetings 👋🏽 

In this PR we present the following:

- Users can add books and will automatically show up in the screen.
- Each book has a unique id.
- Users can remove books from the list.
- Adding a book will create a collection of books by id.
- Removing a book will delete from the array.

### CODE REVIEW

File: scripts/script.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const books = [];
+/* eslint-disable */
+function addBooks() {
```
Comment (rloterh) [inline scripts/script.js:3]: - [ ] Added data in your app is not being persisted to the memory of the browser. Kindly ensure that data is preserved in the browser's memory by using `localStorage`.

File: README.md [hunk 2]
```diff
@@ -1,76 +1,55 @@
...
-
-### Run tests
-
-### Deployment
-
+- Terminal (Bash)
+- Git
+- JavaScript
 
+### Setup: How to start the TicTacToe game
```
Comment (rloterh) [inline README.md:18]: - [ ] OPTIONAL
You have used an unsuitable subheading in your ReadMe. Kindly update this subheading `Setup: How to start the TicTacToe game` to something like `Setup: Getting Started with Awesome-Book app`

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Angel and Chibuike, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors


## Required Changes ♻️
- [ ] _Kindly the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] _Kindly consider my comments marked [OPTIONAL] under this review:_


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ad9311** [comment]
The following changes have been made:
- Added local storage to the script.
- Information will be saved in local storage and render every time the user opens the webpage.
- Both addBook and removeBook functions, add and delete books from localstorage and DOM.

**Review by mimipeshy — APPROVED** [review]
## Hi @Team 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**